api/models/endpoints-models.js

```diff
@@ -0,0 +1,7 @@
+const endpoints = require('../../endpoints.json')
+
+exports.selectAllEndpoints = () => {
```

Comment:
You could avoid this and just send back the json in the controller - you're not doing any database stuff so dont really need the model. Up to you though and if it works, just leave it